src/test/resources/testconfigs/broad.json

```diff
@@ -1,27 +1,27 @@
 {
   "testUsers": [
     {
-      "email": "Penelope.TwilightsHammer@test.firecloud.org",
+      "email": "penelope.twilightsHammer@test.firecloud.org",
```

Comment:
Why are you making these changes? You can just compare ignoring case. I think we should keep these the way they are specified in the Broad's spreadsheet. @zloery do you remember where that is?

Reply:
I requested these changes in an earlier comment on this PR.

Spreadsheet: https://docs.google.com/spreadsheets/d/17PjpwWedN-hxeOlEbepNahLmRDqhvLc25yYxbz6Oj3E/edit#gid=0

I thought it would be cleaner to be lower-case here (one place), versus having to convert to lower case in various tests. Also, people don't normally upper-case emails; ie when they use terra cli, email will be lower-case. I don't think we need to copy spreadsheet exactly. But I don't feel too strongly.

If you change this back to upper-case: Please add a comment in the test:

// Convert to lower-case because email in broad.json is mixed case

If you leave this as lower-case: You forgot to lower-case the "H" in "Hammer".

Reply:
I'm _leaning_ toward not doing this only because we've had bugs where we needed to do lowercase comparisons on emails. You're right that it's very uncommon, but if it's a valid email, we should support it. Changing this file will affect all tests, and it will err on the side of making things pass that should have failed. I'll let Guang be the tie-breaking vote.

I'm not sure where in the stack we lose the case information. It might be Sam, or a GCP API.